core/src/main/java/com/linecorp/armeria/internal/common/ArmeriaHttpUtil.java

```diff
@@ -727,6 +735,45 @@ public static void toArmeria(io.netty.handler.codec.http.HttpHeaders inHeaders,
         if (cookieJoiner != null && cookieJoiner.length() != 0) {
             out.add(HttpHeaderNames.COOKIE, cookieJoiner.toString());
         }
+
+        maybeSetContentLengthUnknown(inHeaders.contains(HttpHeaderNames.CONTENT_LENGTH), out);
+    }
+
+    private static void maybeSetContentLengthUnknown(boolean hasContentLength, HttpHeadersBuilder out) {
```

Comment:
I understood since this method is only called from call paths (network -> user land) this change practically makes no difference in behavior. (for both server, client side content-length is not set unless the remote sent the header)

Let me know if anything did change though.

Reply:
Your understanding is correct. This method is used to call `.contentLengthUnknown()` when converting Netty headers to Armeria headers in decoders.